projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -23,7 +23,7 @@
 class Jetpack_Recommendations {
 
 	const PUBLICIZE_RECOMMENDATION     = 'publicize';
-	const SECURITY_PLAN_RECOMMENDATION = 'security-plan';
+	const SECURITY_PLAN_RECOMMENDATION = 'protect';
```

Comment:
Same question as before, though if its not for reusability, I can see how it would still be useful to go and see the one source of truth "value" in a constant such as this. I'll go ahead and update them 😄